### PR TITLE
Add schemdraw Python library

### PR DESCRIPTION
**Pull request by ByteEOR**
Bucknell University's ECE Department wishes to include schemdraw v0.17 to PrairieLearn. Thank you!

### CODE REVIEW

File: images/plbase/python-requirements.txt [hunk 1]
```diff
@@ -43,3 +43,5 @@ types-Pygments==2.15.0.1
 types-pytest-lazy-fixture==0.6.3.3
 typing-extensions==4.7.0
 tzlocal==4.3
+schemdraw==0.17
+*Add schemdraw:0.17 to PrairieLearn
```
Comment (nwalters512) [inline images/plbase/python-requirements.txt:47]: Why is this line here?

Comment (ByteEOR) [reply images/plbase/python-requirements.txt:47]: https://prairielearn.readthedocs.io/en/latest/questionRuntime/#installing-libraries-in-your-course

This line told me to leave a comment on the request... was it the wrong file to leave comment?

Comment (nwalters512) [reply images/plbase/python-requirements.txt:47]: If you're talking about to "At the bottom of the page you can give a descriptive title to your changes", that's referring to the commit message. There should not be any additional changes to the file itself. In this case, you could have replaced "Update python-requirements.txt".

I can see how "at the bottom of the page" is confusing given that GitHub recently changed the in-browser editor for files. I can update those docs to make this clearer for future contributors.

Comment (nwalters512) [reply images/plbase/python-requirements.txt:47]: Updated docs: https://github.com/PrairieLearn/PrairieLearn/pull/8125

Comment (ByteEOR) [reply images/plbase/python-requirements.txt:47]: Thank you very much!

**Review by eliotwrobson — APPROVED** [review]
From some light reading, this library seems really cool! There are some questions in the example course that use hardcoded images of circuit diagrams, would be neat if some of those could be replaced with images generated by this.

EDIT: Made an issue for this #8128
